hashes.json

```diff
@@ -0,0 +1,5 @@
+[
+  "1D-65-40-5A-63-82-77-4F-2E-99-F2-00-B0-59-9B-4D-B3-71-2B-1B-C1-01-8A-4B-D6-02-C4-7A-8B-11-DB-E8",
+  "02-DB-AC-A8-8C-27-4E-A2-8F-A2-7E-D5-92-72-08-9C-F7-5A-DB-B7-5A-88-64-B9-54-05-55-51-7F-6D-56-F6",
+  "78-87-24-5C-9A-40-0A-F0-2B-48-F4-7D-3B-C4-33-3E-64-8B-54-73-20-55-8B-8D-F5-E8-DE-99-E6-43-94-54"
+]
```

Comment:
It's not a huge deal, but it's kind of hard now to tell which hash is for which version. I guess as long as they all work it doesn't really matter though!

Reply:
Since JSON doesn't support comments, I added new elements that are just treated as hashes that will never match anything, so they effectively act like comments. 😄